client-implementation.md

```diff
@@ -0,0 +1,67 @@
+# Client Implementations
+
+This document describes client implementation recommendations for interoperability with registry servers and other clients using the OCI Distribution Spec.
+
+## Backwards Compatibility
+
+Client implementations SHOULD support registries that implement partial or older versions of the OCI Distribution Spec.
+This section describes how client fallback procedures when an API is not available.
+
+### Referrers API
+
+The Referrers API here is described by [Listing Referrers](spec.md#listing-referrers) and [end-12](spec.md#endpoints).
```

Comment:
"Referrers" needs to be explained before this sentence.
Should be also noted that this has nothing to do with HTTP Referers

Reply:
And "Referrers" vs "Refers" ?

Reply:
There are a few places where the Refers field (schema) and Referrers API are used interchangeably.

Reply:
Referrers and Refers have been added to the Definitions